sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1453,6 +1453,38 @@ graph LR
     Recording -->|End Recording| Executing
 ....
 
+==== Transitive Queue Recording
+
+Submitting a command-group to a queue in the executable state can implicitly
+change its state to `queue_state::recording`. This will occur when the
+command-group depends on an event that has been returned by a queue in the
+recording state. The change of state happens before the command-group is
+submitted to the device (i.e. a new graph node will be created for that command-group).
+
+A queue whose state has been set to `queue_state::recording` using this
+mechanism, will behave as if it had been passed as an argument to
+`command_graph::begin_recording()`. In particular, its state will not
+change again until `command_graph::end_recording()` is called.
```

Comment:
Making a note here that we should say something about the properties that we're passing by `begin_recording()`, if they are or aren't passed to the implicitly recorded queue. Tending towards that they are passed to queue being implicitly recording.

Reply:
Added some extra wording for this behaviour.

Reply:
LGTM, thanks